app/Models/BuildPropagationHistory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class BuildPropagationHistory extends Model
+{
+    protected $table = 'osu_build_propagation_histories';
```

Comment:
if only the table is called `build_propagation_histories`, this line isn't even needed.